site/content/en/docs/Legacy Snapshot/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Legacy Snapshot"
+weight: 1
+description: >
+  Legacy docs
+weight: 100
+---
+
+## Legacy Snapshot
+
+Due to historical reasons, almost all of Prow's documentation resides in the
+[kubernetes/test-infra](k/t-i) repo. As part of the [migration
+effort](https://github.com/kubernetes-sigs/prow/issues/4), this page captures
+Prow documentation in the [kubernetes/test-infra](k/t-i) repository as of [May
+31,
+2022](https://github.com/kubernetes/test-infra/tree/28269f0ce52487379134afe7738393557d577a58).
+All Markdown files have been copied in along with their directory structure
+here.
```

Comment:
I think you mentioned your intentions for these files to me, can we capture that here or in the "Contribution Guidelines" page? Specifically when one of the legacy docs needs to be updated how would we like that to happen? Should we move it out of the legacy snapshot directory at that time?

Reply:
> I think you mentioned your intentions for these files to me, can we capture that here or in the "Contribution Guidelines" page?

Done (added instructions to Contribution Guidelines).

> Specifically when one of the legacy docs needs to be updated how would we like that to happen? Should we move it out of the legacy snapshot directory at that time?

Yes, we should move it out of Legacy Snapshot and move it to a more appropriate location (the instructions in the Contribution Guidelines say this now).